Source/NSubstitute/Core/CallResults.cs

```diff
@@ -6,12 +6,18 @@ namespace NSubstitute.Core
     public class CallResults : ICallResults
     {
         readonly ICallInfoFactory _callInfoFactory;
+        private readonly bool _skipVoidMethods;
         ConcurrentQueue<ResultForCallSpec> _results;
 
-        public CallResults(ICallInfoFactory callInfoFactory)
+        public CallResults(ICallInfoFactory callInfoFactory, bool skipVoidMethodsForPerformance)
```

Comment:
(Not related to this one but) In any case, code duplication is much better than the leaky conventional abstraction that just few people know about 😉